pkg/storage/bench_pebble_test.go

```diff
@@ -361,3 +362,53 @@ func BenchmarkBatchBuilderPut(b *testing.B) {
 
 	b.StopTimer()
 }
+
+func BenchmarkCheckSSTConflicts(b *testing.B) {
```

Comment:
I think it'd be nice to include an extra branch to this benchmark for whether or not the SST keys overlap -- in the overlap=true case it is as-is, but in the overlap=false case it'd e.g. put all the SST keys between two existing keys or something, so that the SST's target span is totally empty.